content/news/2024-12-12-mixxing-up-pen-and-paper.md

```diff
@@ -0,0 +1,122 @@
+
+*[Play "Sixteen Strikes" (Baldur's Gate 3)](https://music.youtube.com/watch?v=NiSDtrVphZc)*
+
+### Layered Loops
+What I described in the intro is an example for a layered song. This one has a base melody for the spooky ambiance and one "voice" track on top of that being layered in the closer you get to the cube.
+
+Great soundtracks (for my use case) will have all layers put behind each other in ascending order and allow me to progress trough multiple same-length loops in a song as a scene gets more and more tense. This used to be pretty easy even before Mixxx introduced loop cues.
+
+However when the composers weren't lazy when mixing down these soundtracks for an OST release, like with Portal 2, these are the least usable files for me unfortunately.
+
+*[Play "Covenant Prayers" (Halo 5)](https://music.youtube.com/watch?v=ofWai1rsI9k)*
+
+### Multiple transitioning Loops
+This is the by far most common format of big name game soundtracks by now - and before the introduction of loop cues in 2.4.1, it was mostly unusable for me.
+
+These soundtracks consist of multiple looping themes that will fit each setting in an entire area of a game. There are ambiance bits, there are combat bits, there may be a theme for the town square. The often varying lengths of those loops made it really difficult to use for me. I mean ... still no challenge for a dedicated DJ.
+
+But a storyteller having to allocate their brain to running a pen and paper session alongside mixing? No chance.
+
+Loop Cues however fixed that for me. I can now just set up to eight loops of different sizes within a single audio file and this has made some of my least favorite music pieces to my most favorite.	
+
+*[Play "a Pirate's Life" (Assassin's Creed IV - Black Flag)](https://music.youtube.com/watch?v=AuEXQnH_jOU)*
+
+## So is it worth it?
+Honestly the prep work I put into these soundtracks so far has not always payed out as much as I would have wanted. It is a lot of fun to slice and dice video game songs and build a huge library of potential epic moments in pen and paper.
+
+However as sessions often do not work as planned, trying to always find and think of the perfect music on the fly can be quite challenging. Especially because me fiddling with music is distracting in itself.
+
+I highly recommend creating a playlist for each session and having some general purpose playlists to jump into when the players inevitably drift off. Themes for places or characters (both player and non-player) can also make a big difference and are not too difficult to keep track of.
+
+However I think a dedicated app with a lot more creature comforts and a lot less DJ-specific features would be needed to make this useful for the average DM. But I think that is fine. Because everyone has their own secret sauce for spicing up their campaigns. And mine is music.
```

Comment:
I am not sure if I got the point, but maybe this sentence can be improved because the reader is likely already a Mixxx user. Let me try: 

> Sure, Mixxx's DJ-Ing features are probably overwhelming for an average DM. But once I was used to it, music has become my secret sauce for spicing up my campaigns.

The another aspect you may want to mention is that Mixxx is fully customizable because it is open source. So we may consider to hack up a Role Play skin, or add specific features. I am sure there are some overlaps with Radio DJs, another underrepresented user group among us contributors.

Reply:
Hm. If it is really easy to customize, I would look into creating a dedicated RPG skin and then publish this blogpost. That would make it a lot more useful.

However there are a couple of must-have features for this that I right now have trouble imagining. For instance allowing users to share songs they sliced in some manner would make this a lot easier and that would need to be implemented and curated. Will halt releasing the blogpost until I looked into the theisability of creating a DM skin.

Reply:
> If it is really easy to customize

It could be much more easy though. You can make probably some minor adjustment with ease, but a full DM skin in release quality is a lot of work. A start could be a hand written moc-up or a copy and past screenshot that demonstates the desired changes. The best is to open a GitHub issues for it. 

> For instance allowing users to share songs they sliced in some manner would make this a lot easier and that would need to be implemented and curated.

Do you mean to share a track along with Cue points and loops? We have that as an Experimental Serato Metadata synchronization. You may test it and report issues you find or features that are missing as another GitHub issue.